### PR TITLE
add hub bundle for easy installation on upgraded instances

### PR DESCRIPTION
**Pull request by blizzz**
Tada:

Screenshot_20200123_210802

Q: Why are there so few?
A: https://github.com/nextcloud/server/issues/19104

Included are all that are being offered in the installation assistant on Nextcloud Hub when doing a fresh install.

### CODE REVIEW

**Review by JanDragon — APPROVED** [review]

File: lib/private/App/AppStore/Bundles/HubBundle.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OC\App\AppStore\Bundles;
+
+class HubBundle extends Bundle {
+
+	public function getName() {
+		return (string)$this->l10n->t('Hub bundle');
```
Comment (nickvergessen) [inline lib/private/App/AppStore/Bundles/HubBundle.php]:
```suggestion
		return $this->l10n->t('Hub bundle');
```

**Comment by nickvergessen** [comment]
The hub icon list was 7 icons?

**Comment by blizzz** [comment]
> The hub icon list was 7 icons?

8. I am confused, too :)

On a fresh 18 install from the tar ball, those I added will be installed if you agree to. 

* Files is shipped, of course
* Photos is shipped
* Activity is shipped
* Talk is neither shipped nor offered to install
* contacts, calendar, mail, only office are all covered

I am absolutely eager  to adding Talk, but we should be consistent.

**Comment by rullzer** [comment]
> > The hub icon list was 7 icons?
> 
>     1. I am confused, too :)
> 
> 
> On a fresh 18 install from the tar ball, those I added will be installed if you agree to.
> 
>     * Files is shipped, of course
> 
>     * Photos is shipped
> 
>     * Activity is shipped
> 
>     * Talk is neither shipped nor offered to install
> 
>     * contacts, calendar, mail, only office are all covered
> 
> 
> I am absolutely eager to adding Talk, but we should be consistent.

Yes I noticed this as well. Talk should be on that list. I will create a PR for the install. I could have sworn it was there maybe a force push somewhere removed it.

**Review by skjnldsv — APPROVED** [review]
sure (aside from comment)

**Comment by blizzz** [comment]
> > > The hub icon list was 7 icons?
> > 
> > 
> > ```
> > 1. I am confused, too :)
> > ```
> > 
> > 
> > On a fresh 18 install from the tar ball, those I added will be installed if you agree to.
> > ```
> > * Files is shipped, of course
> > 
> > * Photos is shipped
> > 
> > * Activity is shipped
> > 
> > * Talk is neither shipped nor offered to install
> > 
> > * contacts, calendar, mail, only office are all covered
> > ```
> > 
> > 
> > I am absolutely eager to adding Talk, but we should be consistent.
> 
> Yes I noticed this as well. Talk should be on that list. I will create a PR for the install. I could have sworn it was there maybe a force push somewhere removed it.

And so Talk is added here.

**Pull request merged**